File: src/routes/kpi.js
const routering = require('express').Router();
const currentHostRunsExecutions = require('../controllers/kpi')

routering.get('/prevMonht/kpi',(req,res)=>{
    currentHostRunsExecutions.listarPrevMonht(req,res);
})

routering.get('/lastMonht/kpi',(req,res)=>{
    currentHostRunsExecutions.listarLastMonht(req,res);
})


module.exports = routering

File: src/routes/kpi.py
from flask import Blueprint, request, jsonify
from controllers.kpi import listarPrevMonht, listarLastMonht

routerkpi = Blueprint('routering', __name__)

@routerkpi.route('/prevMonht/kpi', methods=['GET'])
def prev_month_kpi_route():
    return listarPrevMonht(request, jsonify)

@routerkpi.route('/lastMonht/kpi', methods=['GET'])
def last_month_kpi_route():
    return listarLastMonht(request, jsonify)